<launch>
  <node type="visual_skin_node2" pkg="lfd_vision" name="visual_skin_node_slab2" output="screen">
    <param name="pkg_dir" value="$(find lfd_vision)/" />
    <param name="cam_config" value="config/usbcam2fslab21.yaml" />
    <param name="stereo_config" value="config/usbcam2fslab21.yaml" />
    <param name="blobtrack_config" value="config/usbcam2fslab21.yaml" />
    <param name="objdettrack_config" value="config/usbcam2fslab21.yaml" />
    <param name="blob_calib_prefix" value="blob_" />
    <param name="frame_skip" value="1" />
  </node>
</launch>
